packages/preview-middleware-client/src/cpe/changes/validator.ts

```diff
@@ -0,0 +1,30 @@
+import ResourceBundle from 'sap/base/i18n/ResourceBundle';
+
+/**
+ * Function to validate if a given value is a valid binding model.
+ *
+ * @param value value to be checked.
+ */
+export function vaildateBindingModel(value: string): void {
```

Comment:
you have a typo 😉 It should be `validate`

Reply:
Fixed